PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/IDEHelperImpl.java

```diff
@@ -393,6 +409,69 @@ public void openAppServiceFile(final AppServiceFile file, Object context) {
         AzureTaskManager.getInstance().runInModal(task);
     }
 
+    private boolean openFileInEditor(final AppServiceFile appServiceFile, VirtualFile virtualFile, FileEditorManager fileEditorManager) {
+        final FileEditor[] editors = fileEditorManager.openFile(virtualFile, true, true);
+        if (editors.length == 0) {
+            return false;
+        }
+        for (FileEditor fileEditor : editors) {
+            if (fileEditor instanceof TextEditor) {
+                final String originContent = getTextEditorContent((TextEditor) fileEditor);
```

Comment:
if it is for comparing content changes, use md5 is enough

Reply:
Will do in later pr.